guide/C/ch_invest.xml

```diff
@@ -741,20 +741,20 @@ Income
 
         <listitem>
           <para><emphasis>Create the New Security</emphasis> - To use a new stock, you must create the stock as a
-            new commodity
+            new commodity.
           </para>
 
           <note>
-            <para>Be sure to first select Account Type <emphasis>Stock </emphasis> or <emphasis>Mutual Fund</emphasis>
-              so that the <guilabel>Select...</guilabel> button brings up the list of securites
+            <para>Be sure to first select Account Type <emphasis>Stock</emphasis> or <emphasis>Mutual Fund</emphasis>
+              so that the <guibutton>Select...</guibutton> button brings up the list of securities
```

Comment:
There are three points to discuss.
(1) The string literal ...  is used in GnuCash program.
https://hosted.weblate.org/search/gnucash/gnucash/?offset=1&q=...&sort_by=-priority%2Cposition&checksum=

As a result, it is used in guide and help now.
Therefore, if we want to change it, we should change the original GnuCash msgids.

(2) U+2026 HORIZONTAL ELLIPSIS is not in ASCII character space. 
So I don't think it is better to be in msgids.

(3) U+2026 HORIZONTAL ELLIPSIS is used for ommision.
But ... is not used for ommision in GnuCash.
It may have a historical background, but it is not suitable now.

In my opinion, all of ... should be removed from GnuCash msgids and corresponding documents.

Reply:
> In my opinion, all of ... should be removed from GnuCash msgids and corresponding documents.

I think everyone agrees that '...' should be replaced with '…'. It's just a matter of timing because doing so will fuzzy a lot of msgids. We intended to do that last spring in the run-up to 4.0 but it didn't happen, so we now plan to do it for 5.0. We want to defer it until late in the development cycle to avoid problems merging up from maint.

I don't see any reason not to switch immediately to U+2026 in gnucash-docs. The visual appearance is similar enough to '...' that nobody will be confused.

Yes, U+2026 will have to be escaped as "\u2026" in C/C++ code. So what? Most of the labels that use it are in GtkBuilder (Glade) files and those are already encoded in UTF-8, no escapes required.

I don't understand your point about omission. That's about quoting what someone else has said or written, not a common thing to do in a program's strings. There are other usages for ellipses, including the HIG requirement that a menu item that opens a dialog requiring further input to complete the menu action be marked with one at the end.

Reply:
> I don't see any reason not to switch immediately to U+2026 in gnucash-docs. The visual appearance is similar enough to '...' that nobody will be confused.
> Yes, U+2026 will have to be escaped as "\u2026" in C/C++ code. So what? Most of the labels that use it are in GtkBuilder (Glade) files and those are already encoded in UTF-8, no escapes required.

If you, the master developers, think they are OK, I have no objection.
On some legacy environment, non ASCII characters can't exist in msgids. But they are 
already unsupported environments for GnuCash.

 
> I don't understand your point about omission. That's about quoting what someone else has said or written, not a common thing to do in a program's strings. There are other usages for ellipses, including the HIG requirement that a menu item that opens a dialog requiring further input to complete the menu action be marked with one at the end.

I see. That is the language difference between English and Japanese.
What you are saying is included in [HIG](https://developer.gnome.org/hig/stable/writing-style.html.en#ellipses). This was my oversight.

But we translate English messages that contain ellipses into Japanese messages in which ellipses are not necessary.
I think this has caused my misunderstanding.

Reply:
As for Japanese, We can easily use U+2026 because HORIZONTAL ELLIPSIS is from JIS X 1986 and other wchar character codes.